entproto/cmd/protoc-gen-entgrpc/methods.go

```diff
@@ -91,7 +92,7 @@ func (g *serviceGenerator) generateMutationMethod(op string) error {
 			return nil, %(statusErrf)(%(invalidArgument), "invalid argument: %s", err)
 		}`, g.withGlobals(tmplValues{
 			"reqVar":      reqVar,
-			"checkIDFlag": strconv.FormatBool(op == "create"),
+			"checkIDFlag": strconv.FormatBool(op == "update"),
```

Comment:
Some suggestion - maybe use ent.Op instead of raw strings?

```go
op.Is(ent.UpdateOne|ent.Update)
```

Reply:
i'm changing it to be one of this:
https://github.com/ent/contrib/blob/10cfc868a7de57e98b93f4aeb3233ea9d5da1d42/entproto/service.go#L32-L35
ent.Op doesn't have `Get` which is why I created this thing in the first place

Reply:
mm. they are unexported and in another package 🤷